src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -159,20 +201,16 @@ google.auth.getApplicationDefault(function(err, authClient) {
   }
 
   var request = {
-    // TODO: Change placeholders below to appropriate parameter values for the 'delete' method:
+    // Required. The resource name of the log to delete. Example: `"projects/my-project/logs/syslog"`.
+    logName: "projects/{MY-PROJECT}/logs/{MY-LOG}", // TODO: Update placeholder value(s).
 
-    // * Required. The resource name of the log to delete. Example: `"projects/my-project/logs/syslog"`.
-    logName: "projects/{MY-PROJECT}/logs/{MY-LOG}",
-
-    // Auth client
     auth: authClient
   };
 
-  logging.projects.logs.delete(request, function(err, result) {
+  logging.projects.logs.delete(request, function(err) {
     if (err) {
       console.log(err);
-    } else {
-      console.log(result);
+      return;
```

Comment:
Doesn't it alter behavior? if there's an error and we don't return, then the program will log a bad response below